privaterelay/templates/profile.html

```diff
@@ -26,7 +26,7 @@
   {% with request.user.profile_set.first as user_profile %}
     data-premium-subdomain-set="{{ user_profile.subdomain }}"
   {% endwith %}
-  data-premium-enabled="{{ settings.PREMIUM_ENABLED }}"
+  data-premium-enabled="True"
```

Comment:
Not sure how ya feel about TODOs in the code, but I would recommend adding a one-liner about why this is here, and that it should be removed at a later date. 
```suggestion
{# TODO: Remove the `premium-enabled` attribute once https://github.com/mozilla/fx-private-relay-add-on/pull/246 has been merged/deployed   #}
  data-premium-enabled="True"
```

Reply:
Excellent callout, don't know why I didn't do that here.